src/genie/libs/parser/nxos/show_interface.py

```diff
@@ -3276,14 +3303,45 @@ def cli(self, interface, output=None):
                 interface_dict.update({'description': m.groupdict()['description']})
                 continue
 
-            # vpc
+            # vpc ID, for port-channels only
             p15 = re.compile(r'^vpc +(?P<vpc>\S+)$')
```

Comment:
I know you didn't make this parser initially, but if you could move all these `re.compile`s to before the loop that would be greatly appreciated